### ISSUE TITLE
Clicking Run in AI Command window throws an exception and halts.

### ISSUE THREAD

**Issue by m3ltaw4y** [keijiro/AICommand#6]
NullReferenceException: Object reference not set to an instance of an object
AICommand.OpenAIUtil.InvokeChat (System.String prompt) (at Assets/Editor/OpenAIUtil.cs:52)
AICommand.AICommandWindow.RunGenerator () (at Assets/Editor/AICommandWindow.cs:38)
AICommand.AICommandWindow.OnGUI () (at Assets/Editor/AICommandWindow.cs:64)
UnityEditor.HostView.OldOnGUI () (at <603bdbf3494743328394e419389d220c>:0)
UnityEngine.UIElements.IMGUIContainer.DoOnGUI (UnityEngine.Event evt, UnityEngine.Matrix4x4 parentTransform, UnityEngine.Rect clippingRect, System.Boolean isComputingLayout, UnityEngine.Rect layoutSize, System.Action onGUIHandler, System.Boolean canAffectFocus) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.IMGUIContainer.HandleIMGUIEvent (UnityEngine.Event e, UnityEngine.Matrix4x4 worldTransform, UnityEngine.Rect clippingRect, System.Action onGUIHandler, System.Boolean canAffectFocus) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.IMGUIContainer.HandleIMGUIEvent (UnityEngine.Event e, System.Action onGUIHandler, System.Boolean canAffectFocus) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.IMGUIContainer.HandleIMGUIEvent (UnityEngine.Event e, System.Boolean canAffectFocus) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.IMGUIContainer.SendEventToIMGUIRaw (UnityEngine.UIElements.EventBase evt, System.Boolean canAffectFocus, System.Boolean verifyBounds) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.IMGUIContainer.SendEventToIMGUI (UnityEngine.UIElements.EventBase evt, System.Boolean canAffectFocus, System.Boolean verifyBounds) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.IMGUIContainer.ProcessEvent (UnityEngine.UIElements.EventBase evt) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.CallbackEventHandler.HandleEvent (UnityEngine.UIElements.EventBase evt) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.CallbackEventHandler.HandleEventAtCurrentTargetAndPhase (UnityEngine.UIElements.EventBase evt) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.CallbackEventHandler.HandleEventAtTargetPhase (UnityEngine.UIElements.EventBase evt) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.MouseCaptureDispatchingStrategy.DispatchEvent (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.EventDispatcher.ApplyDispatchingStrategies (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel, System.Boolean imguiEventIsInitiallyUsed) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.EventDispatcher.ProcessEvent (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.EventDispatcher.ProcessEventQueue () (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.EventDispatcher.OpenGate () (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.EventDispatcherGate.Dispose () (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.EventDispatcher.ProcessEvent (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.EventDispatcher.Dispatch (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel, UnityEngine.UIElements.DispatchMode dispatchMode) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.BaseVisualElementPanel.SendEvent (UnityEngine.UIElements.EventBase e, UnityEngine.UIElements.DispatchMode dispatchMode) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.UIElementsUtility.DoDispatch (UnityEngine.UIElements.BaseVisualElementPanel panel) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.UIElementsUtility.UnityEngine.UIElements.IUIElementsUtility.ProcessEvent (System.Int32 instanceID, System.IntPtr nativeEventPtr, System.Boolean& eventHandled) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.UIEventRegistration.ProcessEvent (System.Int32 instanceID, System.IntPtr nativeEventPtr) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.UIElements.UIEventRegistration+<>c.<.cctor>b__1_2 (System.Int32 i, System.IntPtr ptr) (at <55377d8dc0ee4d67989deb78b2b4baf8>:0)
UnityEngine.GUIUtility.ProcessEvent (System.Int32 instanceID, System.IntPtr nativeEventPtr, System.Boolean& result) (at <ba905a90b9394e2ab68807c5b21df438>:0)

**Comment by BrendenDev** [comment]
I found I also get this error when the key is invalid. That being said, my key isn't invalid, but I did find that my free credits for the API usage are expired. 

Found at: https://platform.openai.com/account/usage 
<img src="https://user-images.githubusercontent.com/97307416/226215043-12ea708d-7710-4f8b-9b49-bc78e86b5093.png" width="650">

So it's possible your credits for the API usage are expired or depleted.
